auth0/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "Auth0",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "Auth0",
+  "metric_prefix": "auth0.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "View and analyze your Auth0 events.",
+  "guid": "9308a35c-219e-4d24-ac11-af2511e5041a",
+  "support": "core",
```

Comment:
It cannot be core if its not in core repo

Reply:
Updated `support` to `contrib`